src/resolvers/registries/npm-resolver.js

```diff
@@ -67,7 +67,7 @@ export default class NpmResolver extends RegistryResolver {
           name: 'package',
           type: 'list',
           message: config.reporter.lang('chooseVersionFromList', body.name),
-          choices: Object.keys(body.versions).reverse(),
+          choices: (semver: Object).rsort(Object.keys(body.versions)),
```

Comment:
I have no idea why this `(semver: Object)` syntax is needed but without it I get a flow error about property `rsort` not being found in module `semver`. It's weird because on line 49 we call `semver.satisfies` with no issues. 🤷‍♂️

Reply:
The Object annotation is necessary because the flow-typed definition being used for the semver package is old. It [contains a definition for satisfies](https://github.com/yarnpkg/yarn/blob/master/flow-typed/npm/semver_v5.1.x.js#L63), but does not have one for rsort. 

The [latest definition](https://github.com/flowtype/flow-typed/blob/master/definitions/npm/semver_v5.1.x/flow_v0.25.x-/semver_v5.1.x.js#L144) does contain rsort, so you could probably get rid of the Object annotation by updating.

Reply:
That makes sense. I was actually looking for the type definitions but I looked in the semver package itself. I didn't realize all these flow definitions were stored externally.

Reply:
I tried updating the flow definitions for semver and it broke a lot of stuff. Leaving this as is for now.